waggle-dance-api/pom.xml

```diff
@@ -50,11 +50,11 @@
         <version>4.0.3</version>
     </dependency>
 
-    <!-- Hibernate for validation annotations -->
+<!--     Hibernate for validation annotations
```

Comment:
If we don't need this anymore we should remove it. I thought it was needed as described in the comment though?

Reply:
Removed.